<launch>
    <arg name="veh"/>
    <arg name="config_file_name" default="$(arg veh)"/>

    <!-- define the picar machine to run the remote nodes on. "veh" should be the hostname of the car -->
    <machine
            name="$(arg veh)"
            address="$(arg veh).local"
            user="pi"
            timeout="30"
            env-loader="/home/pi/picar_mum/catkin_ws/env.sh"
    >
    </machine>

    <!-- launch motor node -->
    <group ns="$(arg veh)">
        <node machine="$(arg veh)" name="motor_node" pkg="motor" type="motor_node.py" output="screen">
            <param name="config_file_name" value="$(arg config_file_name)"/>
        </node>
    </group>

    <!-- Subscriptions -->
    <!-- "~car_cmd": picar_msgs/CarCmd -->

    <!-- Publications -->

</launch>